docs/api/ShallowWrapper/simulateError.md

```diff
@@ -26,16 +26,28 @@ function Something() {
 }
 
 class ErrorBoundary extends React.Component {
+  static getDerivedStateFromError(error) {
+    return {
+      hasError: true,
+    };
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
```suggestion
    this.state = { hasError: false };
```
(it's a best practice to always initialize every state field in the constructor)